cinema/views.py

```diff
@@ -35,7 +36,6 @@ class GenreViewSet(
 ):
     queryset = Genre.objects.all()
     serializer_class = GenreSerializer
-    authentication_classes = (TokenAuthentication,)
     permission_classes = (IsAdminOrIfAuthenticatedReadOnly,)
```

Comment:
You added this permission class to the `DEFAULT_PERMISSION_CLASSES` in the `settings.py` file. You don't need it here. Check everywhere.